django_amazon_ses.py

```diff
@@ -27,9 +27,9 @@ def __init__(self, fail_silently=False, **kwargs):
 
         """
         super(EmailBackend, self).__init__(fail_silently=fail_silently)
-        access_key_id = getattr(settings, 'AWS_ACCESS_KEY_ID', None)
-        secret_access_key = getattr(settings, 'AWS_SECRET_ACCESS_KEY', None)
-        region_name = getattr(settings, 'AWS_DEFAULT_REGION', 'us-east-1')
+        access_key_id = getattr(settings, 'AWS_SES_ACCESS_KEY_ID', getattr(settings, 'AWS_ACCESS_KEY_ID', None))
```

Comment:
I believe that these lines are failing the Flake8 tests because they are too long.

Reply:
Thanks for the info @hectcastro, I will split them tonight and commit my changes.